### PR TITLE
feat(bulk): Refactor LastPoint type South

### PR DESCRIPTION
**Pull request by kukukk**

### CODE REVIEW

File: src/south/FolderScanner/FolderScanner.class.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ class FolderScanner extends ProtocolHandler {
    * @return {void}
    */
   constructor(dataSource, engine) {
-    super(dataSource, engine)
+    super(dataSource, engine, { supportListen: false, supportLastPoint: true, supportHistory: false })
```
Comment (jfhenon) [inline src/south/FolderScanner/FolderScanner.class.js:18]: should we have a supportLastFile? or supportFile?

**Review by jfhenon — COMMENTED** [review]
see comment for File support mode

File: src/south/FolderScanner/FolderScanner.class.js (hunk 1 above)
Comment (kukukk) [reply src/south/FolderScanner/FolderScanner.class.js:18]: We have **this.handlesPoints** and **this.handlesFiles** set in every South constructor (used in **getStatus())**. We could use those variables to identify whether the given South sends data as values or file and rename **lastPointQuery()** to **lastFileQuery()** for FolderScanner.

Comment (burgerni10) [reply src/south/FolderScanner/FolderScanner.class.js:18]: What about changing supportLastPoint to supportScanMode ? It's more straightforward and point/file abstract.

lastPointQuery would be renamed scanModeQuery.

About the point/file dichotomy, I think we can eventually merge them in the South connectors. I propose we keep it like that for now, I need to think a bit more about it.

**Review by burgerni10 — APPROVED** [review]

**Pull request merged**